src/test/java/org/folio/entlinks/controller/InstanceAuthorityLinkStatisticsIT.java

```diff
@@ -36,9 +41,7 @@ class InstanceAuthorityLinkStatisticsIT extends IntegrationTestBase {
   private static final OffsetDateTime TO_DATE = OffsetDateTime.of(2021, 10, 10, 10, 10, 10, 10, ZoneOffset.UTC);
   private static final Integer LIMIT = 2;
   private static final AuthorityDataStatActionDto STAT_ACTION_DTO = AuthorityDataStatActionDto.UPDATE_HEADING;
-
   private @MockBean AuthorityDataStatRepository authorityDataStatRepository;
```

Comment:
When you mock the repository it's not an integration test. For integration, we are using a real database that is provided by testcontainers.